tests/gemm_library_node.py

```diff
@@ -0,0 +1,143 @@
+import argparse
```

Comment:
You can use `tests/library/gemm_test.py`. If it ends with `_test.py`, it will run automatically